Mathlib/RingTheory/RingHom/Finite.lean

```diff
@@ -237,3 +237,9 @@ theorem RingHom.finite_ofLocalizationSpan : RingHom.OfLocalizationSpan @RingHom.
   simp_rw [Submonoid.map_powers] at hn₂
   use n₂ + n₁
   exact le_iSup (fun x : s => Submodule.span R (sf x : Set S)) r hn₂
+
+instance {R S : Type*} [CommRing R] {P : Ideal R} [CommRing S] [Algebra R S]
+    [Module.Finite R S] [P.IsPrime] :
+    Module.Finite (Localization.AtPrime P)
+    (Localization (Algebra.algebraMapSubmonoid S P.primeCompl)) :=
+  Module.Finite_of_isLocalization R S _ _ P.primeCompl
```

Comment:
```suggestion
  Module.finite_of_isLocalization R S _ _ P.primeCompl
```
but it looks like this name is pre-existing?

Reply:
Yes, but let's fix it.

Reply:
Oh wait sorry, it looks like the intention was to have anonymous dot notation, which would make a lot of sense. It should therefore rather be
```suggestion
  .of_isLocalization R S _ _ P.primeCompl
```

Also I am noticing that the third and fourth arguments are always `_`. Maybe we should make them implicit?

Reply:
Ah good point!

Reply:
It looks really nice now!!